pootle/settings/30-site.conf

```diff
@@ -15,6 +15,8 @@ MANAGERS = ADMINS
 
 DEBUG = False
 
+CANONICAL_URL = ""
```

Comment:
my feeling is that its not really a `POOTLE` thing - but yep it defo needs some docs